src/quo2/components/info/information_box/style.cljs

```diff
@@ -0,0 +1,66 @@
+(ns quo2.components.info.information-box.style
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.theme :as theme]))
+
+(def ^:private themes
+  {:light {:default      {:bg     colors/white
+                          :border colors/neutral-20
+                          :icon   colors/neutral-50
+                          :text   colors/neutral-100}
+           :informative  {:bg     colors/primary-50-opa-5
+                          :border colors/primary-50-opa-10
+                          :icon   colors/primary-50
+                          :text   colors/neutral-100}
+           :error        {:bg     colors/danger-50-opa-5
+                          :border colors/danger-50-opa-10
+                          :icon   colors/danger-50
+                          :text   colors/danger-50}
+           :close-button colors/neutral-100}
+   :dark  {:default      {:bg     colors/neutral-90
+                          :border colors/neutral-70
+                          :icon   colors/neutral-40
+                          :text   colors/white}
+           :informative  {:bg     colors/primary-50-opa-5
+                          :border colors/primary-50-opa-10
+                          :icon   colors/white
+                          :text   colors/white}
+           :error        {:bg     colors/danger-50-opa-5
+                          :border colors/danger-50-opa-10
+                          :icon   colors/danger-50
+                          :text   colors/danger-50}
+           :close-button colors/white}})
+
+(defn get-color
+  [key]
+  (get-in themes [(theme/get-theme) key]))
```

Comment:
hmm, perhaps you can safe us some future work by using theme as a param and passing it in where it's used? 🤔

Reply:
This is the original code still, but I'll adjust as you suggest :) Thanks